src/main/resources/application.yaml

```diff
@@ -70,8 +70,15 @@ security:
 idam:
   s2s-auth:
     url: ${S2S_URL:http://localhost:4502}
+  api:
+    url: ${IDAM_API_URL:http://localhost:5000}
+  client:
+    id: ${IDAM_CLIENT_NAME:lau}
+    secret: ${IDAM_CLIENT_SECRET:12345678}
+    redirect_uri: ${IDAM_CLIENT_REDIRECT_URI:http://localhost:3451/oauth2redirect}
 
 authorised.services: idam_web_admin,lau_idam_frontend
```

Comment:
We should probably add lau_frontend as Bradley is mergin the case and idam UI's. Will need to keep lau_idam_frontend for now though until the UI merge is complete.